### PR TITLE
Add issue 385, and add Bellroy post about open source

### PR DESCRIPTION
**Pull request by michaelwebb76**
Please and thank you

### CODE REVIEW

File: data/newsletter/issue-385.markdown [hunk 1]
```diff
@@ -4,12 +4,10 @@ This is a weekly summary of what's going on in its community.
...
 
-undefined
+- [Celebrating the release of Amazonka 2.0](https://exploring-better-ways.bellroy.com/celebrating-the-release-of-amazonka-2-0.html) How [Bellroy](https://bellroy.com) thinks about open source
 
 ## Jobs
 
-undefined
-
 Trying to hire a Haskell developer?
 You should [advertise with us](https://haskellweekly.news/advertising.html)!
```
Comment (michaelwebb76) [inline data/newsletter/issue-385.markdown:12]: I figured this was a better default than `undefined` but happy to be corrected on that

File: data/newsletter/issue-385.markdown [hunk 2]
```diff
@@ -4,12 +4,10 @@ This is a weekly summary of what's going on in its community.
 
 ## Featured
 
-undefined
+- [Celebrating the release of Amazonka 2.0](https://exploring-better-ways.bellroy.com/celebrating-the-release-of-amazonka-2-0.html) How [Bellroy](https://bellroy.com) thinks about open source
```
Comment (alexeyzab) [inline data/newsletter/issue-385.markdown:7]:
```suggestion
- [Celebrating the release of Amazonka 2.0](https://exploring-better-ways.bellroy.com/celebrating-the-release-of-amazonka-2-0.html) by Mike Webb
  > Today, we at Bellroy are pleased to discuss our role in the the 2.0 release of Amazonka — a vital Haskell package providing bindings for the AWS (Amazon Web Services) infrastructure.
```

**Comment by alexeyzab** [comment]
Added a suggestion to fit better with our current naming/tag line style, but LGTM otherwise!

**Comment by michaelwebb76** [comment]
Thanks @alexeyzab - I've incorporated your suggestion and I'm not precious about the ordering of items in the "Featured" section - it was just the easiest way to resolve the conflict. If you'd like me to flip them around again just let me know.

**Comment by alexeyzab** [comment]
Nice! No worries, I'll merge as-is and then rearrange later once the issue is more filled out.

**Pull request merged**